src/components/eventsTab.tsx

```diff
@@ -316,33 +321,36 @@ const AdvancedEventFilter = (props) => {
 
   return (
     <>
-      <div className="gf-form gf-form--grow">
-        <Tooltip content="click here for more information">
-          <Button
-            variant="secondary"
-            icon="question-circle"
-            onClick={() => setShowHelp(!showHelp)}
+      <FieldSet style={{ marginBottom: 8 }}>
+        <Label
+          title='Advanced filter'
+          htmlFor={`advanced-filter-${query.refId}`}
```

Comment:
shouldn't there be an `id` prop on one of the child elements pointed to by this id?

Reply:
I used `aria-labelledby` instead. The test accesses it by `page.getByLabel(/Code editor container/)`, so it worked